### DISCUSSION TITLE
Assertion Error for negative price values

### DISCUSSION THREAD

**Discussion by DukeBot18** [kernc/backtesting.py#1149] [Ideas]
I am testing on continuous futures contracts and because of that I have negative price values, this is very common. Take a look at HO ticker. I am running into the issue of an assertion error because of my negative values and negative stop loss values associated. This assertion needs to be removed

**Comment by DukeBot18** [comment]
This is within the Trade Class under the following method:
```
    def __set_contingent(self, type, price):
        assert type in ('sl', 'tp')
        assert price is None or 0 < price < np.inf
```

**Comment by chrisgft** [comment]
@kernc I have put the fix in my forked branch. I can add this to your repo if wanted
